### PR TITLE
err.inspect gives more information than err.class

### PR DESCRIPTION
**Pull request by jasnow**
Change "err.class" to "err.inspect" because it give more information regarding the error.

### CODE REVIEW

**Comment by manhhung741** [comment]
What new information do we gain? I think we have error class & error message and that's about the same as inspect.

File: lib/sorbet-rails/model_rbi_formatter.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def initialize(model_class, available_classes)
       # Load all dynamic instance methods of this model by instantiating a fake model
       @model_class.new unless @model_class.abstract_class?
     rescue StandardError => err
-      puts "#{err.class}: Note: Unable to create new instance of #{model_class.name}"
+      puts "#{err.inspect}: Note: Unable to create new instance of #{model_class.name}"
```
Comment (manhhung741) [inline lib/sorbet-rails/model_rbi_formatter.rb]:
```suggestion
      puts "Note: Unable to create new instance of #{model_class.name}. Error: #{err.inspect}"
```

Comment (jasnow) [reply lib/sorbet-rails/model_rbi_formatter.rb]: ### absolutely

**Comment by jasnow** [comment]
Did you run "rake" with the changed code?

**Comment by jasnow** [comment]
For Rails 5.2, run **bin/rails active_storage:install** before **bin/rails db:migrate**
More information at https://github.com/sferik/rails_admin/issues/3014

**Comment by jasnow** [comment]
Answer is:
```
Note: Unable to create new instance of ActiveStorage::Attachment. Error: #<ActiveRecord::StatementInvalid: Could not find table 'active_storage_attachments'>
Note: Unable to create new instance of ActiveStorage::Blob. Error: #<ActiveRecord::StatementInvalid: Could not find table 'active_storage_blobs'>
Note: Unable to create new instance of ActiveStorage::Attachment. Error: #<ActiveRecord::StatementInvalid: Could not find table 'active_storage_attachments'>
Note: Unable to create new instance of ActiveStorage::Blob. Error: #<ActiveRecord::StatementInvalid: Could not find table 'active_storage_blobs'>
```

**Comment by jasnow** [comment]
Can you restart the build to see if it passes? Thanks

**Comment by jasnow** [comment]
So well....

**Pull request closed without merging**